registry.json

```diff
@@ -4832,5 +4832,19 @@
     "telegram": "Your_ADA_Moon",                                          
     "operatorName": "Alex, Devin",                                   
     "github": "arichard5" 
+ },
+ {
+    "ticker": "BAIDU",                                                       
+    "poolId": "c1f5cfd4330339e90ba83a64d269a81cf415d7adab36403e27b910f7", 
+    "social": {                                                            
+      "twitter": "zjavax",
+      "website": "https://zjavax.github.io/cardano_doc/#/",
+      "facebook": "",  
+      "instagram": "",
+      "youtube": "https://www.youtube.com/channel/UCAfHoyK5b8RoQt9ipN0j4YA/videos"
+    },
+    "telegram": "",                                          
```

Comment:
This is a required field. Please supply the telegram username of an individual and not a group or channel.